libmisc/memzero.c

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Is this file needed at all?

Reply:
Yes.  Otherwise, if the compiler decides to not inline some call to memzero() or strzero(), the linker will fail to find the symbol.

Reply:
And if you declare the functions `static inline ...`?

Reply:
Yes, that solves that.

However, I dislike `static inline` because it's inefficient.  For every translation unit that uses the `static inline` function, if there are calls that are not inlined, the compiler must emit a different internal symbol.  With `extern inline` (C99 inline), you only get one non-inline version of each function.

Okay, okay, shadow probably doesn't need blazing speed.  But to me it feels better conceptually, and it serves as a demonstration of how `extern inline` (C99 inline) should be used correctly, which is not known by many C programmers.